src/ethereumProofPointIssuer.ts

```diff
@@ -0,0 +1,275 @@
+import { ProofPointIssueResult } from "./proofPointIssueResult";
+import { ethers } from "ethers";
+import { EthereumAddress, ProofPointId } from "./ethereumProofPointEvent";
+import { ProofPoint } from "./proofPoint";
+import {
+  IpfsStorageProvider,
+  IpfsStorageProviderSettings,
+  StorageProvider,
+} from "./storage";
+import canonicalizeJson from "canonicalize";
+import localISOdt = require("local-iso-dt");
+import { EthereumAddressResolver } from "./ethereumAddressResolver";
+import {
+  EthereumProofPointRegistry,
+  ETHEREUM_PROOF_TYPE,
+} from "./ethereumProofPointRegistry";
+import { EthereumProofPointRegistryRoot } from "./ethereumProofPointRegistryRoot";
+import { RealHttpClient } from "./httpClient";
+
+/**
+ * Ethereum proof point issuer
+ * Can issue, commit and revoke proof points in an @EthereumProofPointRegistry
+ */
+export class EthereumProofPointIssuer {
```

Comment:
Why is an "Issuer" needed? Could it be an operation on the registry? Just curious...

Reply:
Issuing a proof point involves constructing the JSON, storing it in IPFS and writing the has to the registry, so it's kind of three things, at least one of which isn't directly related to the registry.